src/pages/EventPage/styled.ts

```diff
@@ -7,3 +7,7 @@ export const HighlightedBox = styled.div`
   color: ${usingColors((c) => c.ON.SECONDARY)};
   background-color: ${usingColors((c) => c.SURFACE.SECONDARY)};
 `;
+
+export const DescriptionText = styled.div`
```

Comment:
I think it's more semantically correct to use a `p` tag here, but I could be wrong 🤔

Reply:
Users don't necessarily write one paragraph though, so not sure how it matters

Reply:
Ah, that's true. Maybe we should look into postprocessing the description: splitting the text into multiple `p`s with a double new-line as delimiter.

Reply:
We've discussed a markdown editor as well for more complex control and markup, I think we can revisit this when we get started on that. For now the rendered result looks pretty close to what you see in the input field